src/prng/isaac_serde.rs

```diff
@@ -1,4 +1,15 @@
-#[cfg(feature="serde-1")]
+// Copyright 2013 The Rust Project Developers. See the COPYRIGHT
```

Comment:
It's 2018 now; use that or 2017-2018 (since you started working on it last year)